### ISSUE TITLE
New Project ... should be on top in the File > New menu

### ISSUE THREAD

**Issue by laeubi** [eclipse-platform/eclipse.platform.ui#504]
Currently if one contributes a new project wizzard in the menu it is placed before the more generic "Project ..." item, it seems more appropriate to place the generic one first and the specialized ones below:

![grafik](https://user-images.githubusercontent.com/1331477/210344874-633777aa-6c36-4756-b602-ecd0ba712b4c.png)

**Comment by mickaelistria** [comment]
> it seems more appropriate to place the generic one first and the specialized ones below:

I disagree. The goal of perspective specific wizards is that they're a very small set of projects covering the vast majority of projects a user would initiate in the given context. If the perspective and shortcuts are properly defined, user should have about no strong reason to use the generic one, so there is no strong reason to give it priority over other more contextual ones.

**Comment by laeubi** [comment]
> > it seems more appropriate to place the generic one first and the specialized ones below:
> 
> I disagree. The goal of perspective specific wizards is that they're a very small set of projects covering the vast majority of projects a user would initiate in the given context. If the perspective and shortcuts are properly defined, user should have about no strong reason to use the generic one, so there is no strong reason to give it priority over other more contextual ones.

Then it should be hidden at all an the user can choose the "Others..." if he ever has a strong reason... currently it just looks odd to not give preference to the most generic one I can choose (beside that the order seems random e.g. not alphabetical).

**Comment by mickaelistria** [comment]
> Then it should be hidden at all an the user can choose the "Others..." if he ever has a strong reason... 

That's not a bad idea, but in case of the "Resources" perspective which is free of most shortcuts, the Project... entry seems pretty helpful, having just "Other..."` would be less guiding users when they're tryng to get started. One benefit of Project..  over just Other... is that is shows the filtered list of wizards, so it's not so overwhelming.
